pgjdbc/src/test/java/org/postgresql/test/jdbc2/DatabaseMetaDataPropertiesTest.java

```diff
@@ -147,6 +148,46 @@ public void testIdentifiers() throws SQLException {
 
   }
 
+  @Test
+  public void testIsolationLevels() throws SQLException {
+    DatabaseMetaData dbmd = con.getMetaData();
+    assertNotNull(dbmd);
+
+    assertTrue(dbmd.supportsTransactionIsolationLevel(Connection.TRANSACTION_READ_UNCOMMITTED));
+    assertTrue(dbmd.supportsTransactionIsolationLevel(Connection.TRANSACTION_READ_COMMITTED));
+    assertTrue(dbmd.supportsTransactionIsolationLevel(Connection.TRANSACTION_REPEATABLE_READ));
+    assertTrue(dbmd.supportsTransactionIsolationLevel(Connection.TRANSACTION_SERIALIZABLE));
+
```

Comment:
I'd prefer this was in it's own test. We really don't need it, but if we have to have it, can we create a different test for it?

Reply:
I remove it and renamed the test to reflect the smaller scope.